src/qml/BookmarkRenderer.qml

```diff
@@ -38,55 +39,84 @@ Item {
                     y: mapToScreenPosition.screenPoint.y - height
 
                     width: 36
                     height: 36
-                    source: Theme.getThemeVectorIcon("ic_place_white_24dp")
-                    sourceSize.width: 36 * screen.devicePixelRatio
-                    sourceSize.height: 36 * screen.devicePixelRatio
-
-                    ColorOverlay {
-                        anchors.fill: bookmark
-                        source: bookmark
-                        color: Theme.mainColor
-                    }
 
-                    MouseArea {
-                        anchors.fill: bookmark
-                        onClicked: {
-                            displayToast(qsTr('Bookmark: %1').arg(bookmarkRenderer.bookmarkName));
+                    ShapePath {
+                        strokeWidth: 3
+                        strokeColor: "white"
+                        strokeStyle: ShapePath.SolidLine
+                        fillColor: Theme.mainColor
+                        startX: 6
+                        startY: 16
+                        PathArc {
+                            x: 30
+                            y: 16
+                            radiusX: 12
+                            radiusY: 14
                         }
-                        onDoubleClicked: {
-                            bookmarkModel.setExtentFromBookmark(bookmarkModel.index(bookmarkRenderer.bookmarkIndex, 0));
+                        PathArc {
+                            x: 18
+                            y: 36
+                            radiusX: 36
+                            radiusY: 36
                         }
+                        PathArc{
+                            x: 6
+                            y: 16
+                            radiusX: 36
+                            radiusY: 36
+                        }
+                    }
+
+                    Rectangle {
+                        x: 13
+                        y: 9
+                        width: 10
+                        height: 10
+                        color: "white"
+                        radius: 4
+                    }
+
+                    layer.enabled: true
+                    layer.effect: DropShadow {
+                        transparentBorder: true
+                        radius: 8
+                        samples: 25
+                        color: "#99000000"
+                        horizontalOffset: 0
+                        verticalOffset: 0
                     }
                 }
 
-                Glow {
+                MouseArea {
                     anchors.fill: bookmark
-                    source: bookmark
-                    radius: 7
-                    samples: 17
-                    color: "#44000000"
+                    onClicked: {
+                        displayToast(qsTr('Bookmark: %1').arg(bookmarkRenderer.bookmarkName));
+                    }
+                    onDoubleClicked: {
```

Comment:
Can you open a pr or issue at qfield docs regarding click etc behavior of bookmarks?

Reply:
Will do.